mitmproxy/proxy/layers/http/_http_h2.py

```diff
@@ -49,9 +49,19 @@ class BufferedH2Connection(h2.connection.H2Connection):
 
     def __init__(self, config: h2.config.H2Configuration):
         super().__init__(config)
+        self.local_settings.initial_window_size = 2**24
```

Comment:
How did you end up picking `2**24`? Our only reaction to reaching that limit is to increase it again, so why don't we set it to `2**31-1` right away? :)

(for inbound frame size it's different because that affects multiplexing - here we don't want to go totally over board)

Reply:
> How did you end up picking 2**24?

I kept increasing until it could saturate my connection, which happened around `2**20`, but I noticed curl sets its default at approximately `2**23`, so I decided to increase it to `2**24` for good measure.